### ISSUE TITLE
getstuff.bat doesn't work anymore

### ISSUE THREAD

**Issue by Zelex** [Genoil/cpp-ethereum#183]
ethdev.com looks down or gone

**Comment by Genoil** [comment]
i still have all the stuff locally, but github won't let me add it because of size restrictions. will have to look for an alternative. open to suggestions.

**Comment by douchecanoe** [comment]
btw when you download the files manually you need to change set eth_server to localhost

**Comment by zhou304** [comment]
I've changed eth_server to localhost, run getstuff.bat OK. But run "cmake -DBUNDLE=cudaminer -G "Visual Studio 12 2013 Win64" .." failed. 
Some error messages as following:
-- Could NOT find readline (missing:  READLINE_INCLUDE_DIR READLINE_LIBRARY)
-- Could NOT find miniupnpc (missing:  MINIUPNPC_INCLUDE_DIR MINIUPNPC_LIBRARY)
-- Could NOT find gmp (missing:  GMP_INCLUDE_DIR GMP_LIBRARY)

**Comment by mascondante** [comment]
@zhou304 [MinGW-w64](https://sourceforge.net/projects/mingw-w64/) might give you what you need though that's total speculation. I'm downloading it now to give it a whirl. It has a lot of the core stuff needed for cross-compiling. And it's the only one with a 64 bit option that seems to be actively maintained.

**Comment by zhou304** [comment]
@mascondante I'm trying to build it with MinGW now.

**Comment by mascondante** [comment]
@zhou304 it was a no-go for me. I'm not even remotely an expert at compiling and troubleshooting. Just a passing hobby. If you figure it out please let me know. I see that readline and GMP do exist for Windows in some form googling around but I'm not sure why miniupnpc would be not found considering it's in the packages referenced by @Genoil earlier. I had an error compiling in the Visual Studio phase my first go-round. I noticed I overlooked one of the packages I was supposed to copy over. I've done that and now I'm getting ready to try again. I'll let you know how it goes. Sorry for the disconnected thinking. It's been a while chipping away at this and I've got some burnout going on in my head.

**Comment by zhou304** [comment]
@mascondante ok

**Comment by Phaceial** [comment]
Hey where are we supposed to move the files we downloaded from that link to?

**Comment by mascondante** [comment]
@Phaceial in a folder named Download under extdep. If this folder does not yet exist go ahead and create it. If this is a fresh clone off Github there is no need to edit the getstuff.bat file. Just create the subdirectory Download and place your items there. It will see they already exist and continue to the next step.

@zhou304 after the MinGW step and making sure everything was up to date in VS I tried compiling again. It freezes at the first step and refuses to stop or provide any details as to what is wrong.

`1>------ Build started: Project: ZERO_CHECK, Configuration: Debug x64 ------
1>  Checking Build System
1>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.CppCommon.targets(170,5): warning MSB4220: Waiting for the currently executing task "CustomBuild" to cancel.
`
That is all I get. Perhaps I should not have tried two separate potential solutions at once before testing as I'm not sure what caused the issue but I will say since my very first attempt was with VS 2015 and the same behavior occured I am leaning towards the updates I downloaded from Tools/Extensions and Updates to be the cause in some way. I got further without taking any updates besides the core Update 5. I would advise against optional updates as a potential resolution.
